src/test/java/com/synopsys/integration/alert/performance/LargeNotificationTest.java

```diff
@@ -91,6 +92,32 @@ public void init() {
             .orElse(DEFAULT_NUMBER_OF_PROJECTS_TO_CREATE);
     }
 
+    @Test
+    @EnabledIfEnvironmentVariable(named = "ALERT_RUN_PERFORMANCE", matches = "true")
+    void createTestProjects() throws IntegrationException {
```

Comment:
This is good.  What I found out is that the trigger of the notifications adds a component to the project.  If we don't have  components the performance tests won't work if we rely on the policy to trigger a bunch of notifications.  So we may need to have a method to add components to the projects as well.  A bit out of scope for this PR but something to be aware of.

Reply:
So the original test we used looped twice, once to find/create the project, and once to find/create **then** trigger. I will adjust this to do that find/create then trigger.

Reply:
@psantos1113 I updated the code to do the create + trigger in one loop. Run time went from 67 minutes per 2K -> 97 minutes per 2K. And that is with 5 simultaneous jobs each making 2K.